### ISSUE TITLE
Show all Appear steps in presentation mode next slide preview

### ISSUE THREAD

**Issue by busypeoples** [jxnblk/mdx-deck#76]
The next slide preview should show all steps revealed to be useful.

**Comment by jxnblk** [comment]
What do you mean by *preview* slide?

**Comment by busypeoples** [comment]
That next slide preview in presentation mode.

**Comment by busypeoples** [comment]
Where would you put this context information? Inside `withSlide`. We would need to know if the current Component is a preview slide.

**Comment by vlsi** [comment]
Ok, it looks like  mdx-deck 2 has the following in `Presenter.js`:

```
<Zoom zoom={1 / 4}>
 {Next && (
       <Slide>
           <Next/>
        </Slide>
 )}
</Zoom>
```

I've verified that adding `step={1}` does allow to specify next animation which is nice.
It should be not that hard to add a couple of `if` to compare current `step` with the total steps in slide and pick `slides[index + 1]` vs `slides[index]` accordingly.

**Comment by jxnblk** [comment]
Fixed in #323
